### PR TITLE
Chrome DevTools lesson: edit exact wording from Chrome and change to the same list order

### PR DESCRIPTION
**Pull request by ManonLef**
Complete the following REQUIRED checkboxes:

-   [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [X] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [X] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [X] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

I made these changes for consistency. The wording and order in the TOP lesson now reflects the wording and order from the [Chrome DevTools website](https://developer.chrome.com/docs/devtools/) to avoid unnecessary scrolling back and forth. 

Since this lesson also only uses the Chrome Developer Tools, I changed some wording to specifically mention Chrome. 

I also removed an unnecessary sublist item for opening the Developer Tools.

**2. This PR:**

- Remove Opening Developer Tools options list 1.1 and change it into list item 1.
- change "Open DevTools" to "Open Chrome Devtools"
- edit assignment list order
- add 'Chrome' to make it obvious the lesson is talking about Chrome Developer Tools. 

**3. Additional Information:**

Discord:
https://discord.com/channels/505093832157691914/540903304046182425/949789924247347280

### CODE REVIEW

File: foundations/javascript_basics/javascript_developer_tools.md [hunk 1]
```diff
@@ -34,15 +33,15 @@ There are three ways to open the Developer Tools menu:
 Google has updated some of the required sections in the below tutorials and some elements have changed or no longer exist, but you can still follow along using the same functionality and tools that they cover. For example, you may be asked to inspect a button that is no longer on the page- you can still follow along and inspect existing elements without issue.
 
 1. Head to the [Chrome DevTools Documentation](https://developers.google.com/web/tools/chrome-devtools/) by Google. The following subsections cover what you'll be using the Developer Tools for 95% of the time.  Feel free to skip the elements you are already familiar with:
```
Comment (01zulfi) [inline foundations/javascript_basics/javascript_developer_tools.md:35]: The chrome developer tools link redirects to a different URL. We should update the current link to https://developer.chrome.com/docs/devtools/

```suggestion
1. Head to the [Chrome DevTools Documentation](https://developer.chrome.com/docs/devtools/) by Google. The following subsections cover what you'll be using the Developer Tools for 95% of the time.  Feel free to skip the elements you are already familiar with:
```

**Review by 01zulfi — CHANGES_REQUESTED** [review]
Thanks for doing this. I have a requested change below. After that we should be good to merge.

File: foundations/javascript_basics/javascript_developer_tools.md (hunk 1 above)
Comment (ManonLef) [reply foundations/javascript_basics/javascript_developer_tools.md]: Update as requested

**Comment by ManonLef** [comment]
I initially hit "resolve conversation" on the suggested change. Not sure if I should have done that. I'm still a bit new around here and tried to google who's the common 'resolver' but apparently opinions differ.

Would love to hear what is common in this case or for the curriculum 

I unresolved it to let @01zulfi decide if it was satisfactorily resolved

**Comment by 01zulfi** [comment]
@ManonLef There isn't a general rule regarding resolving conversations here. From what I've seen in the curriculum repository, contributors usually mark conversations as resolved themselves after they've made the requested change.

Thanks for asking and thanks for making these changes as well! :fire:

**Review by 01zulfi — APPROVED** [review]

**Pull request merged**